### PR TITLE
blog(embertimes154): ember-stargate

### PR DESCRIPTION
**Pull request by chrisrng**

### CODE REVIEW

File: source/2020-06-26-the-ember-times-issue-154.md [hunk 1]
```diff
@@ -9,19 +9,34 @@ responsive: true
...
+```
+
+Application:
+```hbs
+{{outlet}}
+
+<footer class="sticky-footer">
+  <PortalTarget @name="footer-title" class="sticky-footer__title" />
+</footer>
+```
```
Comment (ijlee2) [inline source/2020-06-26-the-ember-times-issue-154.md:37]:
```suggestion
The `ember-stargate` addon aims to solve this problem without using private APIs, Ember's `component` helper, or having to jump through hoops to implement. All you need to make a portal work is to define the content that should be sent through the portal using `<Portal @target="some-target">`, and the target - identified by its name - where it should be rendered to using `<PortalTarget @name="some-target">`.

For example, suppose you want to update the footer in `application.hbs` from a child route:

Route:

```hbs
<Portal @target="footer-title">Step 1</Portal>
```

Application:

```hbs
{{outlet}}

<footer class="sticky-footer">
  <PortalTarget @name="footer-title" class="sticky-footer__title" />
</footer>
```
```

File: source/2020-06-26-the-ember-times-issue-154.md [hunk 2]
```diff
@@ -9,19 +9,34 @@ responsive: true
...
 
-<add your name to author list, top and bottom>
-<add blurb and emoji to "SOME-INTRO-HERE">
+“Portals” are a way to render things in a different place of the DOM tree than they are logically defined in the app. There are a number of existing solutions in the Ember ecosystem for the same problem such as [ember-wormhole](https://github.com/yapplabs/ember-wormhole), [ember-elsewhere](https://github.com/ef4/ember-elsewhere), the native `{{in-element}}` (previously using the [ember-in-element-polyfill](https://github.com/ember-polyfills/ember-in-element-polyfill)).
+
+The `ember-stargate` addon aims to solve this problem without using private APIs, Ember's `component` helper, or having to jump through hoops to implement. All you need to make a portal work is to define the content that should be sent through the portal using `<Portal @target="some-target">`, and the target - identified by its name - where it should be rendered to using `<PortalTarget @name="some-target">`. For example, you have 2 templates the first being the route template and the second being the `application.hbs`:
```
Comment (ijlee2) [inline source/2020-06-26-the-ember-times-issue-154.md]:
```suggestion
The `ember-stargate` addon aims to solve this problem without using private APIs, Ember's `component` helper, or having to jump through hoops to implement. All you need to make a portal work is to define the content that should be sent through the portal using `<Portal @target="some-target">`, and the target - identified by its name - where it should be rendered to using `<PortalTarget @name="some-target">`.

For example, suppose you want to update the footer in `application.hbs` from a child route:
```

File: source/2020-06-26-the-ember-times-issue-154.md [hunk 3]
```diff
@@ -9,19 +9,34 @@ responsive: true
...
+
+Route:
```
Comment (ijlee2) [inline source/2020-06-26-the-ember-times-issue-154.md:27]:
```suggestion
Route:

```

File: source/2020-06-26-the-ember-times-issue-154.md [hunk 4]
```diff
@@ -9,19 +9,34 @@ responsive: true
...
+```hbs
+<Portal @target="footer-title">Step 1</Portal>
+```
+
+Application:
```
Comment (ijlee2) [inline source/2020-06-26-the-ember-times-issue-154.md:33]:
```suggestion
Application:

```

**Review by ijlee2 — APPROVED** [review]
Looks good to me, Chris. Thanks for the writeup!

The 1st suggestion is optional, but the other two (adding a new line) are required to get linting to pass.

**Comment by chrisrng** [comment]
> Looks good to me, Chris. Thanks for the writeup!
> 
> The 1st suggestion is optional, but the other two (adding a new line) are required to get linting to pass.

Thanks!

**Pull request merged**